src/SettingView.ts

```diff
@@ -21,14 +21,28 @@ import DigitalGardenSiteManager from "./DigitalGardenSiteManager";
 import { SvgFileSuggest } from "./ui/file-suggest";
 import { addFilterInput } from "./ui/addFilterInput";
 
+interface IObsidianTheme {
+	name: string;
+	author: string;
+	screenshot: string;
+	modes: string[];
+	repo: string;
+	legacy: boolean;
+}
+
+const OBSIDIAN_THEME_URL =
+	"https://raw.githubusercontent.com/obsidianmd/obsidian-releases/master/community-css-themes.json";
 export default class SettingView {
 	private app: App;
-	private settings: DigitalGardenSettings;
-	private saveSettings: () => Promise<void>;
```

Comment:
the addFilterInput calls these directly, so they should probably not be private?